### PR TITLE
Julia StarterKit initial commit

### PR DESCRIPTION
**Pull request by Arkoniak**
Starter Kit for Julia 0.6

External Dependency on Memento logger.

### CODE REVIEW

File: airesources/Julia/hlt/game.jl [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    width::Int
+    height::Int
+
+    function Game(botname::String)
+        id = get_msg()
+        setup_logger(botname, id)
+        botname = @sprintf("%s-%s", botname, id)
+
+        width, height = map(x -> parse(Int, x), split(get_msg()))
+        send_msg(botname, true)
```
Comment (Snaar) [inline airesources/Julia/hlt/game.jl]: Can you address https://github.com/HaliteChallenge/Halite-II/issues/151?

Comment (Arkoniak) [reply airesources/Julia/hlt/game.jl]: Fixed.

File: airesources/Julia/MyBot.jl [hunk 2]
```diff
@@ -17,8 +17,12 @@ using Memento
 logger = get_logger("bot-logger")
 
 game = Game("Settler")
+
+# Here one can do all preliminary checks and warmups, using initial 60s timeout.
```
Comment (Snaar) [inline airesources/Julia/MyBot.jl]: Can you have example of how to get the game_map here for initial inspection? Cpp bot for example prints basic map info such as player and planet counts.

Comment (Arkoniak) [reply airesources/Julia/MyBot.jl]: Yes, no problem at all. Added example.

**Comment by Snaar** [comment]
Oh, feel free to come up with some other default name just for Julia that is not "Settler" :P

**Pull request merged**